lavalink/Cargo.toml

```diff
@@ -15,23 +15,23 @@ repository = "https://github.com/twilight-rs/twilight.git"
 version = "0.6.0"
 
 [dependencies]
-tokio-tungstenite = { default-features = false, features = ["connect"], version = "0.14" }
 dashmap = { default-features = false, version = "4.0" }
 futures-util = { default-features = false, features = ["bilock", "std", "unstable"], version = "0.3" }
 http = { default-features = false, optional = true, version = "0.2" }
 tracing = { default-features = false, features = ["std", "attributes"], version = "0.1" }
 percent-encoding = { default-features = false, optional = true, version = "2" }
 serde = { default-features = false, features = ["derive", "std"], version = "1" }
-serde_json = { default-features = false, features = ["alloc"], version = "1" }
-tokio = { default-features = false, features = ["net", "rt", "time"], version = "1.0" }
+serde_json = { default-features = false, features = ["std"], version = "1" }
+tokio = { default-features = false, features = ["macros", "net", "rt", "sync", "time"], version = "1.0" }
```

Comment:
Same here, why were `macros` and `sync` enabled?

Reply:
They are used when connecting to a node (`src/node.rs`)